Error due to a new field "cpuFrequency"

The component is not able to handle the field cpuFrequency from a NodeCandidate when trying to deploy an app.

```
{"@timestamp":"2024-12-03T11:31:57.951745555Z","@version":"1","message":"Default on delivery for delivery for topic://eu.nebulouscloud.cfsb.get_node_candidates_multi.reply","logger_name":"eu.nebulouscloud.exn.core.Consumer","thread_name":"pool-1-thread-9","level":"DEBUG","level_value":10000}
{"@timestamp":"2024-12-03T11:31:57.965235138Z","@version":"1","message":"Processing message fortopic://eu.nebulouscloud.cfsb.get_node_candidates_multi.reply","logger_name":"eu.nebulouscloud.exn.core.Consumer","thread_name":"pool-1-thread-9","level":"DEBUG","level_value":10000}
Exception in thread "Thread-0" java.lang.IllegalArgumentException: Unrecognized field "cpuFrequency" (class org.ow2.proactive.sal.model.Hardware), not marked as ignorable (10 known properties: "state", "owner", "cores", "location", "id", "providerId", "disk", "ram", "fpga", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: org.ow2.proactive.sal.model.NodeCandidate["hardware"]->org.ow2.proactive.sal.model.Hardware["cpuFrequency"])
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4624)
	at com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:4555)
	at eu.nebulouscloud.optimiser.controller.ExnConnector.lambda$findNodeCandidatesMultiple$3(ExnConnector.java:640)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source)
	at eu.nebulouscloud.optimiser.controller.ExnConnector.findNodeCandidatesMultiple(ExnConnector.java:643)
	at eu.nebulouscloud.optimiser.controller.NebulousAppDeployer.deployApplication(NebulousAppDeployer.java:435)
	at eu.nebulouscloud.optimiser.controller.NebulousApp.deploy(NebulousApp.java:737)
	at eu.nebulouscloud.optimiser.controller.ExnConnector$PerformanceIndicatorMessageHandler.lambda$onMessage$0(ExnConnector.java:359)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "cpuFrequency" (class org.ow2.proactive.sal.model.Hardware), not marked as ignorable (10 known properties: "state", "owner", "cores", "location", "id", "providerId", "disk", "ram", "fpga", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: org.ow2.proactive.sal.model.NodeCandidate["hardware"]->org.ow2.proactive.sal.model.Hardware["cpuFrequency"])
	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
	at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1153)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2241)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1793)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperties(BeanDeserializerBase.java:1743)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:467)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1493)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:348)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185)
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:545)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeWithErrorWrapping(BeanDeserializer.java:576)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:446)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1493)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:348)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185)
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4619)
	... 14 more
{"@timestamp":"2024-12-03T11:32:00.194359079Z","@version":"1","message":"Processing scheduled executor [health] topic://eu.nebulouscloud.optimiser_controller.health  ","logger_name":"eu.nebulouscloud.exn.core.Manager","thread_name":"pool-1-thread-8","level":"DEBUG","level_value":10000}
```

Tested with prod and dev env.

--- a/nebulous-requirements-extractor/build.gradle
+++ b/nebulous-requirements-extractor/build.gradle
@@ -38,7 +38,9 @@ dependencies {
     api 'com.fasterxml.jackson.core:jackson-databind:2.17.2'
 
     // SAL client library; exported to consumers
-    api 'org.ow2.proactive:sal-common:13.1.0-SNAPSHOT'
+    api('org.ow2.proactive:sal-common:13.1.0-SNAPSHOT') {
+        changing = true
+    }
 
     // YAML parsing: https://github.com/FasterXML/jackson-dataformats-text
     implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.17.2'

--- a/optimiser-controller/build.gradle
+++ b/optimiser-controller/build.gradle
@@ -46,7 +46,9 @@ dependencies {
     implementation 'info.picocli:picocli:4.7.6'
 
     // SAL client library
-    implementation 'org.ow2.proactive:sal-common:13.1.0-SNAPSHOT'
+    implementation('org.ow2.proactive:sal-common:13.1.0-SNAPSHOT') {
+        changing = true
+    }
 
     // Logging: SLF4J (implemented by Logback)
     implementation 'ch.qos.logback:logback-classic:1.5.6'
@@ -55,8 +57,9 @@ dependencies {
 
     // the EXN Middleware:
     // https://openproject.nebulouscloud.eu/projects/nebulous-collaboration-hub/wiki/asynchronous-messaging-specification
-    implementation ('eu.nebulouscloud:exn-connector-java:1.0-SNAPSHOT') {
+    implementation('eu.nebulouscloud:exn-connector-java:1.0-SNAPSHOT') {
         exclude group: 'org.slf4j', module: 'slf4j-simple'
+        changing = true
     }
 
     // Analysing KubeVela

